packages/mdc-button/README.md

```diff
@@ -68,6 +68,14 @@ We recommend you load [Material Icons](https://material.io/icons/) from Google F
   <i class="material-icons mdc-button__icon">favorite</i>
   Button
 </button>
+
```

Comment:
I would split this out to its own code block and introduce it with a paragraph e.g. "It's also possible to use an SVG icon:"

Also, RE the actual SVG code, we could potentially hollow it out to only the parts that matter for purposes of the README, e.g.:

```html
<button class="mdc-button">
  <svg class="mdc-button__icon" xmlns="http://www.w3.org/2000/svg" viewBox="...">
    ...
  </svg>
  Button
</button>
```

(In particular, I'm not sure why `fill="#000000"` is in there right now, wouldn't that override our CSS??)

Reply:
Will do. 

`fill=...` is an attribute style, and not inline styling (`style="fill:...`). CSS styles always override attribute styles, because `fill=...` is converted to a css style with a specificity of 0.